agentops/helpers.py

```diff
@@ -30,6 +30,21 @@ def getinstance(*args, **kwargs):
     return getinstance
 
 
+def conditional_singleton(class_):
```

Comment:
Had to do something similar in #257, in case a conflict emerges

Reply:
I think we should remove if we're not using